CMake/ConfigureCrossCompile.cmake

```
@@ -0,0 +1,42 @@
+# This file adds the necessary configurations to cross compile
+# mlpack for embedded systems. You need to set the following variables
+# from the command line: the CMAKE_SYSROOT, TOOLCHAIN_PREFIX and the
+# board type.
+# This file will compile OpenBLAS if it is downloaded and it is not
+# available on you system in order to find the BLAS library.
+
+if (CMAKE_CROSSCOMPILING)
+  include(board/flags-config.cmake)
+  if(NOT CMAKE_SYSROOT AND (NOT TOOLCHAIN_PREFIX))
+    message(FATAL_ERROR "Neither of CMAKE_SYSROOT or TOOLCHAIN_PREFIX is set, please set both of them and try again")
+  elseif(NOT CMAKE_SYSROOT)
+    message(FATAL_ERROR "Can not proceed CMAKE_SYSROOT is not set")
+  elseif(NOT TOOLCHAIN_PREFIX)
+    message(FATAL_ERROR "Cant not proceed TOOLCHAIN_PREFIXN is not set")
+  elseif(NOT OPENBLAS_TARGET)
+    message(FATAL_ERROR "Cant not proceed, board name is not set, please refer to documentation")
+  endif()
+endif()
+
+macro(search_openblas version)
+  set(BLA_STATIC ON)
+  find_package(BLAS)
+  if (NOT BLAS_FOUND OR (NOT BLAS_LIBRARIES))
```

```suggestion
  if(NOT BLAS_FOUND OR (NOT BLAS_LIBRARIES))
```

Let's use the same style across the file.